core/src/main/java/com/scalar/db/storage/jdbc/JdbcDatabaseAdmin.java

```diff
@@ -690,6 +707,24 @@ private void createTableInternal(
               + " INITRANS 3 MAXTRANS 255";
       execute(connection, alterTableStatement);
     }
+
+    if (hasDescClusteringOrder
+        && (rdbEngine == RdbEngine.POSTGRESQL || rdbEngine == RdbEngine.ORACLE)) {
+      // For PostgreSQL and Oracle, create a unique index for the clustering orders
+      String createUniqueIndexStatement =
+          "CREATE UNIQUE INDEX "
```

Comment:
Just a question.
So, for PostgreSQL and Oracle, key ordering is not really supported and we need to use a secondary index to use desc ordering?

Reply:
Yes, that's correct.